stickyuploads/storage.py

```diff
@@ -12,4 +15,18 @@ def __init__(self):
 
     def url(self, name):
         """Files are not accessable via url."""
-        raise NotImplementedError()
\ No newline at end of file
+        raise NotImplementedError()
+
+    def get_available_name(self, name, max_length=None):
+        """Return path to name placed in two random directories"""
+        name = os.path.join(
```

Comment:
Because I want number of files/directories in a directory to be relatively small. I remember when we have a problem with speed in a site where there was too many (tents of thousands) files in a single directory. Probably the reason was that it was with ext3 file system but still.
There need to be a tool for clearing old files (and probably empty directories) and if the list with all directories is too big there can be problems.